torch/CMakeLists.txt

```diff
@@ -267,11 +267,7 @@ target_compile_definitions(torch PUBLIC _THP_CORE)
 
 # until they can be unified, keep these lists synced with setup.py
 if(MSVC)
-  if (${CMAKE_BUILD_TYPE} MATCHES "Debug")
-    set (MSVC_RUNTIME_LIBRARY_FLAG "/MDd")
-  else()
-    set (MSVC_RUNTIME_LIBRARY_FLAG "/MD")
-  endif()
+  set (MSVC_RUNTIME_LIBRARY_FLAG "/MD$<$<CONFIG:Debug>:d>")
```

Comment:
Looks like this variable is not used anywhere, would you please change it to `MSVC_RUNTIME_LIBRARY_OPTION` which it should be?

Reply:
If it's not used then we can just remove it?
To be honest, I personally remove all this custom MT/MD stuff modification in my fork as it makes no sense. They get inherited anyway.

Reply:
I'm okay with removing them since they are useless here.